tecken/tests/test_sentry.py

```diff
@@ -226,3 +231,34 @@ def test_count_sentry_scrub_error(metricsmock):
     metricsmock.assert_incr(
         "tecken.sentry_scrub_error", value=1, tags=["host:testnode"]
     )
+
+
+def test_sentry_wrap_non_app_error_has_release():
+    port = os.environ.get("EXPOSE_SENTRY_PORT", 8090)
+
+    # Flush fakesentry to ensure we fetch only the desired error downstream
+    requests.post(f"http://fakesentry:{port}/api/flush/")
+
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    base_dir = os.path.join(os.path.dirname(current_dir), "..")
+    release = get_release_name(base_dir)
```

Comment:
We might want to assert that the release is in an expected shape. Namely something like:

* it's "unknown", OR
* value has a ":" in it

But this is good enough for now.

Reply:
There are also tests for `get_release_name` already in `test_libdockerflow.py` which test that sort of thing.